src/retrieve-data-helpers/launch-context-retrieval.js

```diff
@@ -0,0 +1,50 @@
+/* eslint no-param-reassign: 0 */
+import axios from 'axios';
+
+function retrieveLaunchContext(link, accessToken, patientId, fhirBaseUrl) {
+  return new Promise((resolve, reject) => {
+    const headers = {
+      Accept: 'application/json',
+      Authorization: `Bearer ${accessToken.access_token}`,
+    };
+
+    const launchParameters = {
+      patient: patientId,
+    };
+
+    if (link.appContext) {
+      launchParameters.appContext = link.appContext;
+    }
+
+    // May change when the launch context creation endpoint becomes a standard endpoint for all EHR providers
+    axios({
+      method: 'post',
+      url: `${fhirBaseUrl}/_services/smart/Launch`,
```

Comment:
just for my information, why using _ prefix here?

Reply:
also, why cap on the L?

Reply:
The default FHIR server for this Sandbox is HSPC. This was just the endpoint that they decided to use for registering a launch context for this scenario.